portfolio/src/main/webapp/index.html

```diff
@@ -22,8 +22,8 @@ <h1>Gabi Newman's Portfolio</h1>
       </p>
       <p>This page is pink because that is Gabi's favorite color.</p>
       <p>Here is a fun selfie she took a while ago!</p>
-      <img id="selfie" src="/images/selfie.jpg" alt="An image of Gabi posing with
-          her hand on her face."/>
+      <img id="home-selfie" src="/images/selfie.jpg" alt="An image of Gabi posing with
+          her hand on her face." onclick="changeColor()"/>
```

Comment:
Is this supposed to do something? I didn't notice anything when I clicked

Reply:
Oh, oops! I changed the image id on this picture to avoid a bug where it had the same id as a picture in the gallery. I must have added the onclick method by accident, though. It's gone now.